WhaleSpotting/react-app/src/tests/Card.test.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import Card from "../components/Card";
+import SightingApiModel from "../apiModels/SightingApiModel";
+
+const exampleConfirmed: SightingApiModel = {
+    sightedAt: new Date(),
+    species: "orca",
+    quantity: 3,
+    location: "Sea",
+    longitude: 1.232,
+    latitude: 2.312,
+    description: "Whales at sea",
+    orcaType: "Orca",
+    orcaPod: "",
+    confirmed: true,
+    userId: 2,
+    username: "FakeUserConfirmed"
+};
+
+const exampleUnconfirmed: SightingApiModel = {
+    sightedAt: new Date(),
+    species: "orca",
+    quantity: 3,
+    location: "Sea",
+    longitude: 1.232,
+    latitude: 2.312,
+    description: "Whales at sea",
+    orcaType: "Orca",
+    orcaPod: "",
+    confirmed: false,
+    userId: 2,
+    username: "FakeUserNotConfirmed"
+};
+
+describe("Card tests", () => {
+    test("Renders card", () => {
+        render(<Card sighting={exampleConfirmed}/>);
+        const cardComponent = screen.getByTestId("card-component");
+        expect(cardComponent).toBeInTheDocument();
+    });
+
+    test("Does not render pending for confirmed card", () => {
+        render(<Card sighting={exampleConfirmed}/>);
+        const pending = screen.getByTestId("confirmed");
```

Comment:
Should this be called confirmed? Also this te3st doesn't test what you are saying, it tests that the confirmed is rendered

Reply:
fixed